Tests/ReactiveSwiftTests/SignalProducerSpec.swift

```diff
@@ -793,6 +793,14 @@ class SignalProducerSpec: QuickSpec {
 				expect(dates) == [tick1, tick2, tick3]
 			}
 
+			it("shouldn't overflow on a real scheduler") {
+				let scheduler = QueueScheduler()
+				let producer = timer(interval: 3, on: scheduler)
```

Comment:
True dat, I thought it was > 9, but it's actually > 2 that causes issues. Will update title.